src/DatasetTree.ts

```diff
@@ -109,7 +109,15 @@ export class DatasetTree implements vscode.TreeDataProvider<ZoweNode> {
                 }
             } else if (favoriteSearchPattern.test(line)) {
                 const sesName = line.substring(1, line.lastIndexOf("]")).trim();
-                const zosmfProfile = Profiles.getInstance().loadNamedProfile(sesName);
```

Comment:
I'm ok with this. However, it makes me worry since in the above `if` clause, we enclosed every zowe operation as well as the node creation with the same try-catch in case we failed to create the session for any reason.